sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/SendSmsToGroupExample.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.sms.samples.quickstart;
+
+import com.azure.communication.sms.SmsClient;
+import com.azure.communication.sms.SmsClientBuilder;
+import com.azure.communication.sms.models.SmsSendOptions;
+import com.azure.communication.sms.models.SmsSendResult;
+import com.azure.core.util.Context;
+import java.util.Arrays;
+
+public class SendSmsToGroupExample {
+    public static void main(String[] args) {
+        String connectionString = System.getenv("COMMUNICATION_CONNECTION_STRING");
+        String phoneNumber = System.getenv("COMMUNICATION_PHONE_NUMBER");
+        SmsClient smsClient = new SmsClientBuilder()
+            .connectionString(connectionString)
+            .buildClient();
+
+        SmsSendOptions options = new SmsSendOptions();
+        options.setDeliveryReportEnabled(true);
+        options.setTag("Marketing");
+
+        Iterable<SmsSendResult> sendResults = smsClient.sendWithResponse(
+            phoneNumber,
+            Arrays.asList(phoneNumber),
```

Comment:
Can we put in more than 1 phone number? This doesn't really send to a group and may be confusing

Reply:
to not use more env. variables I just used one phone number.